tests/system_tests/wandb_backend_spy/spy.py

```diff
@@ -192,6 +192,31 @@ def history(self, *, run_id: str) -> dict[int, Any]:
             history_parsed[offset] = json.loads(line)
         return history_parsed
 
+    def output(self, *, run_id: str) -> dict[int, Any]:
+        """Returns the history file for the run.
+
+        The file is represented as a dict that maps integer offsets to
+        JSON objects.
+
+        Args:
+            run_id: The ID of the run.
+
+        Raises:
+            KeyError: if the run does not exist.
+        """
+        spy = self._assert_valid()
+
+        try:
+            run = spy._runs[run_id]
+        except KeyError as e:
+            raise KeyError(f"No run with ID {run_id}") from e
+
+        output_file = run._file_stream_files.get("output.log", {})
+        output_parsed: dict[int, Any] = {}
+        for offset, line in output_file.items():
+            output_parsed[offset] = line
+        return output_parsed
```

Comment:
The lines aren't JSON objects, so they should be returned as-is.

Also, I'm realizing the type annotation on `_RunData._file_stream_files` is wrong: it should be `dict[str, dict[int, str]]`. With that change, this can be:

```python
return dict(run._file_stream_files.get("output.log", {}))
```